src/test/java/com/epam/tc/homework2/Exercise1Test.java

```diff
@@ -0,0 +1,189 @@
+        driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(40));
+    }
+
+    @AfterMethod
+    public void tearDown() {
+        driver.quit();
+    }
+
+    @Test
+    public void testOpenPage() {
+        SoftAssertions softly = new SoftAssertions();
+
+        //
+        driver.manage().window().maximize();
+        // 1.  Open test site by URL
+        driver.get("https://jdi-testing.github.io/jdi-light/index.html ");
+        // 2. Assert Browser title "Home Page"
+        softly.assertThat(driver.getTitle()).isEqualTo("Home Page");
+        //   Find login's form and click
+        driver.findElements(By.className("navbar-right")).get(0).click();
+        // 3. Perform login
+        driver.findElement(By.id("name")).sendKeys("Roman");
+        driver.findElement(By.id("password")).sendKeys("Jdi1234");
+
+        // Click to bottom Enter
+        driver.findElement(By.id("login-button")).click();
+        // 4.Assert Username is logging
+        String txtUserName = driver.findElement(By.id("user-name")).getText();
+        softly.assertThat(txtUserName).isEqualTo("ROMAN IOVLEV");
+        // 5.Assert that there are 4 items on the header section are displayed,
+        // and they have proper texts( element= Home)
+
+        String fieldHome = driver.findElement(
+            By.xpath("//ul[@class=\"uui-navigation nav navbar-nav m-l8\"]//*[contains(text(),'Home')]")).getText();
+        softly.assertThat(fieldHome).isEqualTo("HOME");
+
+        // 5. Assert that there are 4 items on the header section are displayed,
+        // and they have proper texts ( element = CONTACT FORM )
+
+        String fieldContactForm = driver.findElement(
+                                            By.xpath("//ul[@class=\"uui-navigation nav navbar-nav m-l8\"]"
+                                                + "//*[contains(text(),'Contact form')]"))
+                                        .getText();
+        softly.assertThat(fieldContactForm).isEqualTo("CONTACT FORM");
+
+        // 5. Assert that there are 4 items on the header section are displayed,
+        // and they have proper texts ( element = SERVICE )
+
+        String fieldService = driver.findElement(
+            By.xpath("//ul[@class=\"uui-navigation nav navbar-nav m-l8\"]//*[contains(text(),'Service')]")).getText();
+        softly.assertThat(fieldService).isEqualTo("SERVICE");
+        // 5.Assert that there are 4 items on the header section are displayed,
+        // and they have proper texts ( element = METALS & COLORS )
+        // "METALS & COLORS");
+        String fieldMetalColor = driver.findElement(
+                                           By.xpath("//ul[@class=\"uui-navigation nav navbar-nav m-l8\"]"
+                                               + "//*[contains(text(),'Metals & Colors')]"))
+                                       .getText();
+        softly.assertThat(fieldMetalColor).isEqualTo("METALS & COLORS");
```

Comment:
also, this looks a bit strange for me, bc your xpath already contains verifications of element text. So, this verification should be refactored to.